cleanup/mavenSnapshotCleanupWhenRelease/MavenSnapshotCleanupWhenReleaseTest.groovy

```diff
@@ -0,0 +1,65 @@
+import static org.jfrog.artifactory.client.ArtifactoryClient.create
+
+import org.jfrog.artifactory.client.Artifactory
+
+import groovyx.net.http.HttpResponseException
+import spock.lang.Specification
+
+class MavenSnapshotCleanupWhenReleaseTest extends Specification {
+
+    def TEST_FOLDER = 'org/jfrog/test'
+    def TEST_NAME = 'test-snapshot-cleanup-when-release'
+
+    def 'Maven Snapshot Cleanup When Release test'() {
+        setup:
+        def baseurl = 'http://localhost:8088/artifactory'
+        def artifactory = create(baseurl, 'admin', 'password')
+
+        deleteNoErrorIfNotFound(artifactory, 'maven-local-lib-releases', TEST_FOLDER)
```

Comment:
@axel3rd your test case is assuming the repositories exist. It should create the repositories as part of the setup phase and remove them at cleanup.

You can do that using the following snippet:
```groovy
def builder = artifactory.repositories().builders()
def local = builder.localRepositoryBuilder().key('maven-local')
        .repositorySettings(new MavenRepositorySettingsImpl()).build()
artifactory.repositories().create(0, local)
```